synfig-core/src/synfig/distance.cpp

```diff
@@ -111,11 +111,15 @@ Distance::operator=(const synfig::String& str)
 }
 
 synfig::String
-Distance::get_string(int digits)const
+Distance::get_string(int digits, bool remove_trailing_zeroes_, bool force_decimal_point, bool has_unit)const
```

Comment:
> remove_trailing_zeroes_

Underscore in suffix usually used in private members, so better not use it for arguments.

Reply:
I did so to avoid the naming conflict with the remove_trailing_zeroes() method, I guess I could just rename the variable though.

Reply:
removed altogether as trailing zeroes shouldn't be needed anyway.